### PR TITLE
Add initial draft of "Take Out the Garbage"

### PR DESCRIPTION
**Pull request by nickswalker**
Addresses #493

### CODE REVIEW

File: tasks/TakeOutGarbage.tex [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	\item \textbf{Location:} This test takes place in the arena and its immediate exterior.
+	\item \textbf{Interior Bins:} There are three bins in the apartment for trash, recycling and compost.
+	\item \textbf{Exterior Bins:} There are three larger bins outside the apartment. They may have lids covering their contents.
+\end{itemize}
+
+
+\subsection{Additional rules and remarks}
+\begin{enumerate}[nosep]
+	\item \textbf{Door:} The team decides whether the apartment door starts open or closed.
+	\item \textbf{Exterior bin lids:} If the robot is unable or has failed to open an exterior bin lid, it must clearly say so. The robot may then leave the bag or interior bin next to the exterior bin for a reduction of 30\% in scoring for that type of garbage.
```
Comment (maximest-pierre) [inline tasks/TakeOutGarbage.tex]: I feel that if the robot are unable or failed to open the exterior bin lid the robot should ask the operator to open the lid. They should still have a penalty but less severe than 30%.

File: general_rules/Scenario.tex [hunk 2]
```diff
@@ -56,6 +56,9 @@ \subsection{Furniture}
 	\item a couch,
 	\item a small table,
 	\item a small dinner table with two chairs,
+	\item a trash bin,
+	\item a recycling bin,
+	\item a compost bin,
```
Comment (kyordhel) [inline general_rules/Scenario.tex]: This is a No-Go from my perspective. We are requesting two relatively big and expensive objects per arena to the LOC for a single, non-mandatory test. I'd just drop them and add only the trash bin. Maybe two small trash bins are OK.

The robot can take the kitchen and bathroom trash outside, to the appropriate designated location.

File: tasks/TakeOutGarbage.tex [hunk 3]
```diff
@@ -0,0 +1,61 @@
+\section{Take Out the Garbage [House Holder]}
+The robot has to take out the trash, recycling and compost.
+
+% \subsection{Focus}
+% This test focuses on object perception, manipulation, and planning.
+
+\subsection{Main Goal}
+All garbage in the apartment has has been deposited into the correct exterior bin.
+
+\noindent\textbf{Reward:} 450pts (150pts per type of garbage)
```
Comment (kyordhel) [inline tasks/TakeOutGarbage.tex]: Not sure if 150pts is enough reward for taking out a plastic bag from a bucket on the floor. It has never been done before.

Just 150pts for taking out, opening a lid, and placing inside is completely out of the question. Not even I am that cheap with scoring.

File: tasks/TakeOutGarbage.tex [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+\subsection{Bonus rewards}
+\begin{enumerate}[nosep]
+	\item Opening the apartment door (100pts)
+	\item Closing the apartment door (100pts)
+\end{enumerate}
```
Comment (kyordhel) [inline tasks/TakeOutGarbage.tex]: Threre's something missing here. The test should round up to 1000pts including all bonuses

**Comment by maximest-pierre** [comment]
Can we change it so there is trash around the apartment and the robot must be able to sort and drop it in the right container? This seems more entertaining than seeing a robot move a bags across the arena and as a bonus, we could make the robot bring the bins outside of the arena.

**Comment by kyordhel** [comment]
> Can we change it so there is trash around the apartment and the robot must be able to sort and drop it in the right container?

We already have that task. Check the Clean-Up test in https://github.com/RoboCupAtHome/RuleBook/blob/2019/restructure/tasks/CleanUp.tex

**Comment by nickswalker** [comment]
@kyordhel I think the changes address the bulk of your concerns. This version would allow teams to score the majority of points for taking the pre-sealed trash bags from two buckets in the arena to a collection zone somewhere inside the arena.

As I can see so far, the purpose of the bonus points distinction is to reward teams that accomplish challenging and desirable-to-end-user extensions of the task. From this simplified version of the task, my personal most desired capability would be for the robot to go the last mile and put the trash outside the door so I don't have to manipulate the trash myself. Also important, but potentially significantly more challenging, would be replacing the bag (fine grained manipulation of a deformable object that probably can't be accomplished without bimanipulation). I see no reason not to include both of these as options for ambitious teams. I haven't thought through from where the robot would obtain the bag or other details, so I haven't included that in these changes.

File: general_rules/Scenario.tex (hunk 2 above)
Comment (nickswalker) [reply general_rules/Scenario.tex]: Agree. Moved to a minimum of a single bin which may be small (though a second reduces the shuffling that the ref has to do).

File: tasks/TakeOutGarbage.tex (hunk 3 above)
Comment (nickswalker) [reply tasks/TakeOutGarbage.tex]: Bumped to 300pts for taking a single bag out and putting it on a designated spot on the floor somewhere in the arena.

File: tasks/TakeOutGarbage.tex (hunk 4 above)
Comment (nickswalker) [reply tasks/TakeOutGarbage.tex]: Ah, I read 3.6.3 "The maximum total score per task in Stage I is 1000 points." as, for each task, the scoring of that task is no more than 1000 points. The correct reading is for each task, the maximum attainable score is equal to 1000pts.

File: tasks/TakeOutGarbage.tex (hunk 1 above)
Comment (nickswalker) [reply tasks/TakeOutGarbage.tex]: I agree that this is too much to ask for too little return, so the latest revision drops the lids altogether.

**Comment by justinhart** [comment]
This looks like it works. We can always edit the tasks later to tweak, but it belongs in the competition and the current format is about what was asked for.

**Pull request merged**